includes/editor-templates/navigator.php

```diff
@@ -2,12 +2,31 @@
 namespace Elementor;
 
 use Elementor\Core\Editor\Editor;
+use Elementor\Utils;
+use Elementor\Core\Utils\Promotions\Filtered_Promotions_Manager;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
 
 $is_editor_v2_active = Plugin::$instance->experiments->is_feature_active( Editor::EDITOR_V2_EXPERIMENT_NAME );
+$promotion_data = [
+	'text' => esc_html__( 'Access all Pro widgets', 'elementor' ),
+	'url_label' => esc_html__( 'Upgrade Now', 'elementor' ),
+	'url' => 'https://go.elementor.com/go-pro-structure-panel/',
+];
```

Comment:
`$promotion_data` can be declared but never read. Can you also declare it inside the `if ( ! $has_pro ) {`